eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -136,6 +141,9 @@ jobs:
           $packageOverrides = @()
           $packageOverrides += (dir env: | Where-Object { $_.Name.StartsWith("SMOKE_PACKAGE_") }).Value
```

Comment:
Is this needed still now that it is just one package.

Reply:
Is there a new way that the version is being pinned to the package release @ckairen? If not I think this is still necessary. See [here](https://github.com/Azure/azure-sdk-for-js/blob/966e87e8717e03c7ebf6a8bf86b7d6adc88eeef8/common/smoke-test/Initialize-SmokeTests.ps1#L145).

Reply:
No, not that I know of